merge_test.go

```diff
@@ -15,11 +15,12 @@
 package nutsdb
 
 import (
-	"github.com/stretchr/testify/require"
-	"github.com/xujiajun/utils/strconv2"
 	"sync"
 	"testing"
 	"time"
+
+	"github.com/stretchr/testify/require"
```

Comment:
Why only added dependency in this file? Is that expected?

Reply:
I don't know, maybe my editor did it, I'll check it out